### DISCUSSION TITLE
nil query parameter shows up as string "<nil>" in server

### DISCUSSION THREAD

**Discussion by PeideAmonNg** [oapi-codegen/oapi-codegen#1237] [Q&A]
Hi, when a *string type query parameter is not set, when the request is made, it is showing up in the server as `<nil>` string value as opposed to just nil. 

example req.URL.RawQuery on API caller side:  `pagination%5Btoken%5D=%3Cnil%3E`

client generated by oapi-codegen:
```
type GetBooksParams struct {
	Pagination *Pagination `json:"pagination,omitempty"`
}

type Pagination = PaginationQuery

type PaginationQuery struct {
	Token  *string `json:"token"`
}
```

Any ideas? Thanks in advance

**Comment by PeideAmonNg** [comment]
@jamietanna any idea about this one?

**Reply by jamietanna** [reply]
Mind sharing a minimal spec + configuration I can use to reproduce?
